Packs/Cisco-umbrella-cloud-security/Integrations/Cisco-umbrella-cloud-security/Cisco-umbrella-cloud-security.py

```diff
@@ -68,18 +81,31 @@ def remove_domain(client: Client, **args) -> str:
 
 
 def get_destination_domains(client: Client, **args) -> CommandResults:
-    organizationId = args.get('orgId')
-    destinationListId = args.get('destId')
-    uri = f'/{organizationId}/destinationlists/{destinationListId}/destinations'
+    r = client.get_destinations(args.get('orgId'), args.get('destId'))
 
-    r = client._http_request('GET', uri)
+    results = CommandResults(
+        outputs_prefix="Umbrella.Destinations",
+        outputs_key_field="id",
+        outputs=r.get('data'),
+        readable_output=tableToMarkdown('Domains in Destination List', r.get('data'))
+    )
+
+    return results
 
-    for x in r.get('data'):
-        x['Destination List'] = destinationListId
+
+def get_destination_domain(client: Client, **args) -> CommandResults:
+    r = client.get_destinations(args.get('orgId'), args.get('destId'))
+
+    destination_domain = None
+    for destination in r.get('data'):
+        if destination.get('destination') == args['domain']:
+            destination_domain = destination
 
     results = CommandResults(
-        outputs_prefix="Umbrella.Destinations", outputs_key_field="data.id", outputs=r,
-        readable_output=tableToMarkdown('Domains in Destination List', r.get('data'))
+        outputs_prefix="Umbrella.Destination",
```

Comment:
any reason not to output to `get_destination_lists`? that way we can keep both commands outputting to the same path

Reply:
I'm okay with `get_destination_domain` outputting to `Umbrella.Destinations` to stay consistent with `get_destination_domains`